docs/components_page/components/dropdown.md

```diff
@@ -47,6 +53,12 @@ Use the `active` and `disabled` arguments of `DropdownMenuItem` to set the activ
 
 {{example:components/dropdown/content.py:dropdown}}
 
+## AutoClose
```

Comment:
This property doesn't appear to be working.

Fixing it will likely introduce a breaking change because currently dismissal of the dropdown menu is controlled by the `toggle` prop of each `DropdownMenuItem`.

I think the `auto_close` interface is nicer, but not backwards compatible so... 🤷‍♂️